# MPC formulation setting
dt: 0.05
N: 20

# MAV dynamics param
mass: 1.56
thrust_scale: 26.8 # 26.75, 26.8
thrust_bias: 1.57  # 1.57
roll_time_constant: 0.1591
roll_gain: 1.1443
pitch_time_constant: 0.1336
pitch_gain: 1.0655
drag_coefficient_x: 0.001
drag_coefficient_y: 0.001

# dynamic config default values
q_x : 80
q_y : 80
q_z : 80

q_vx : 8
q_vy : 8
q_vz : 10

r_roll  : 100
r_pitch : 100
r_thrust : 1

roll_max: 25.0        # deg
pitch_max: 25.0       # deg
thrust_min: 0.5       # m*g
thrust_max: 1       # m*g

K_yaw: 1.8
yawrate_max: 90.0     # deg

takeoff_height: 1.2 # m
